proxy/frontend/core/src/main/java/org/apache/shardingsphere/proxy/frontend/ShardingSphereProxy.java

```diff
@@ -89,8 +123,12 @@ private void accept(final List<ChannelFuture> futures) throws InterruptedExcepti
     }
     
     private void createEventLoopGroup() {
-        bossGroup = Epoll.isAvailable() ? new EpollEventLoopGroup(1) : new NioEventLoopGroup(1);
-        workerGroup = getWorkerGroup();
+        if (null == bossGroup) {
```

Comment:
EventLoopGroup could be created once and used in both ServerBootstrap. Please avoid null check here.